js/NavMainComponent.js

```diff
@@ -86,11 +149,27 @@ NavMainComponent.prototype.changeActiveSecondaryNavLink = function(target) {
     component.$html.find('[href="' + target + '"]').addClass('is-active');
 };
 
+NavMainComponent.prototype.changeActiveQuaternaryNavLink = function(target) {
+
+    var component = this;
+
+    component.$html.find('.nav-quaternary .nav-item.is-active').removeClass('is-active');
```

Comment:
as above, we have a reference to the `.nav-quaternary` node list, do we _need_ this duplicate query? Or can we just filter the existing list in memory?